### PR TITLE
Added OrderProcessing as Shopware parameter

### PR DESCRIPTION
**Pull request by adi-stefan**

### CODE REVIEW

File: backend/de.metas.externalsystem/src/main/sql/postgresql/system/80-de.metas.externalsystem/5703180_sys_added_OrderProcessing_Shopware_param.sql [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+INSERT INTO AD_Ref_List_Trl (AD_Language,AD_Ref_List_ID, Description,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Ref_List_ID, t.Description,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Ref_List t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Ref_List_ID=543558 AND NOT EXISTS (SELECT 1 FROM AD_Ref_List_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Ref_List_ID=t.AD_Ref_List_ID)
+;
+
+-- Reference: Order Processing
+-- Value: I
+-- ValueName: Auftrag, Lieferung und Rechnun
+-- 2023-09-18T10:56:40.554252500Z
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Reference_ID,AD_Ref_List_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,541831,543559,TO_TIMESTAMP('2023-09-18 13:56:40.433','YYYY-MM-DD HH24:MI:SS.US'),100,'D','Y','Auftrag, Lieferung und Rechnun',TO_TIMESTAMP('2023-09-18 13:56:40.433','YYYY-MM-DD HH24:MI:SS.US'),100,'I','Auftrag, Lieferung und Rechnun')
+;
+
```
Comment (mrie04) [inline backend/de.metas.externalsystem/src/main/sql/postgresql/system/80-de.metas.externalsystem/5703180_sys_added_OrderProcessing_Shopware_param.sql:73]: Small Typo here: ``Rechnun -> Rechnung`` . Pls fix here and in model and also pls in general set the isTranslated flags for `de_DE`, `de_CH` and `en_US`

**Review by mrie04 — CHANGES_REQUESTED** [review]
Pls check my comment and fix the failling java tests

File: backend/de.metas.externalsystem/src/main/sql/postgresql/system/80-de.metas.externalsystem/5703180_sys_added_OrderProcessing_Shopware_param.sql (hunk 1 above)
Comment (adi-stefan) [reply backend/de.metas.externalsystem/src/main/sql/postgresql/system/80-de.metas.externalsystem/5703180_sys_added_OrderProcessing_Shopware_param.sql:73]: done, except for the isTranslated flag.

**Review by mrie04 — APPROVED** [review]
lgtm 👍

**Pull request merged**